easybuild/easyconfigs/a/anvio/anvio-7-intel-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,215 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonBundle'
+
+name = 'anvio'
+version = '7'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://merenlab.org/software/anvio/'
+description = """An analysis and visualization platform for 'omics data."""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+
+github_account = 'merenlab'
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('Pysam', '0.15.3'),
+    ('matplotlib', '3.1.1', versionsuffix),
+    ('prodigal', '2.6.3'),
+    ('Biopython', '1.75', versionsuffix),
+    ('h5py', '2.10.0', versionsuffix),
+    ('HMMER', '3.2.1'),
+    ('LLVM', '9.0.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('threadpoolctl', '2.1.0', {
+        'source_urls': [
+            'https://files.pythonhosted.org/packages/f7/12/'
+            'ec3f2e203afa394a149911729357aa48affc59c20e2c1c8297a60f33f133'
+        ],
+        'source_tmpl': 'threadpoolctl-2.1.0-py3-none-any.whl',
+        'unpack_sources': False,
+        'checksums': ['38b74ca20ff3bb42caca8b00055111d74159ee95c4370882bbff2b93d24da725'],
+        'use_pip': True,
+    }),
+    ('ipython_genutils', '0.2.0', {
+        'checksums': ['eb2e116e75ecef9d4d228fdc66af54269afa26ab4463042e33785b887c628ba8'],
+    }),
+    ('jupyter_core', '4.6.3', {
+        'checksums': ['394fd5dd787e7c8861741880bdf8a00ce39f95de5d18e579c74b882522219e7e'],
+    }),
+    ('traitlets', '4.3.3', {
+        'checksums': ['d023ee369ddd2763310e4c3eae1ff649689440d4ae59d7485eb4cfbbe3e359f7'],
+    }),
+    ('nbformat', '5.0.6', {
+        'checksums': ['049af048ed76b95c3c44043620c17e56bc001329e07f83fec4f177f0e3d7b757'],
+    }),
+    ('toposort', '1.5', {
+        'checksums': ['dba5ae845296e3bf37b042c640870ffebcdeb8cd4df45adaa01d8c5476c557dd'],
+    }),
+    ('llvmlite', '0.33.0', {
+        'checksums': ['9c8aae96f7fba10d9ac864b443d1e8c7ee4765c31569a2b201b3d0b67d8fc596'],
+    }),
+    ('numba', '0.50.1', {
+        'checksums': ['89e81b51b880f9b18c82b7095beaccc6856fcf84ba29c4f0ced42e4e5748a3a7'],
+    }),
+    ('scikit-learn', '0.23.1', {
```

Comment:
There is no version requirement for `scikit-learn` in `anvio` v7. In such a case, we should use a regular dependency on `scikit-learn` v0.21.3 instead of bundling v0.23.1. In the same way that it was done for anvio v6.1. Or is  v0.21.3 causing issues with `anvio` v7?